### PR TITLE
fix rmsprop learning rate for convergence

### PR DESCRIPTION
**Pull request by murasaki0110**
Rmsprop with default learning rate (0.001) cannot converge in this example. 
Initialize learning rate to (0.0001) and add weight decay fix the problem.

### CODE REVIEW

**Comment by fchollet** [comment]
It does converge, and it did in fact converge many thousands of times since the script was written. Since you don't provide any supporting evidence for the benefits of your proposed changes, I must close this PR.

**Comment by murasaki0110** [comment]
https://github.com/fchollet/keras/issues/5239
As seen in this issue also, the loss will decrease at the begining and start increasing after few epochs due to too large learning rate.
Solution to this is decreasing the learning rate with rmsprop as I supposed or changing the optimizer to Adam or SGD.
